### PR TITLE
removed duplicate file

### PR DESCRIPTION
**Pull request by JustDoom**
# Description

removed duplicate file
not meant to fix any grammar mistakes if there are any

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [x] I have run linting on my branch before submitting the PR

### CODE REVIEW

**Review by RemagOfficial — APPROVED** [review]

File: info/en_us/PLUGINS/USEFUL_PLUGINS.md [hunk 1]
```diff
@@ -59,38 +63,40 @@ The plugins in this category have a focus on fun aspects that are added to the s
...
 
-Dynmap gives players the ability to move around their world through an internet browser
+Dynmap gives players the ability to move around their world through an internet browser.
 
 ---
 
 ## Creativity Plugins
 
-The plugins in this category have a focus on additions that allow players to express their creativity the most.
+The plugins in this category have a focus on additions that allow players to express thier creativity the most.
```
Comment (Silverarmor) [inline info/en_us/PLUGINS/USEFUL_PLUGINS.md:85]:
```suggestion
The plugins in this category have a focus on additions that allow players to express their creativity the most.
```

Comment (Silverarmor) [reply info/en_us/PLUGINS/USEFUL_PLUGINS.md:85]: It was spelt correctly before.

**Review by Silverarmor — APPROVED** [review]
LGTM. One minor fix however.

File: info/en_us/PLUGINS/USEFUL_PLUGINS.md (hunk 1 above)
Comment (JustDoom) [reply info/en_us/PLUGINS/USEFUL_PLUGINS.md:85]: i dont even know how that happened lol

**Pull request merged**